vpp-manager/startup/startup.go

```diff
@@ -201,6 +203,15 @@ func parseEnvVariables(params *config.VppManagerParams) (err error) {
 		}
 	}
 
+	params.TapMtu = DefaultTapMtu
```

Comment:
For vpp-manager we should use the MTU that we find on the uplink interface originally instead of configuring it through an env var. You should modify getInterfaceConfigFromLinux to grab the MTU from link, save it in the InterfaceConfig struct, and then use that to configure the tap.